vpn-ca/main.go

```diff
@@ -173,8 +185,13 @@ func main() {
 	}
 	flag.Parse()
 
+	caDir := filepath.Join(*currentDir, "ca")
```

Comment:
it is confusing to add "ca" without allowing the user to "override" this, it would mean there is always a "ca" directory (created)...

Reply:
removed